src/OrleansDependencyInjection/OrleansDependencyInjection.csproj

```diff
@@ -28,6 +30,7 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DocumentationFile>bin\Release\OrleansDependencyInjection.XML</DocumentationFile>
```

Comment:
You only defined it for Release. I don't know how much it matters but, for example, Orleans.xml is defined for both Release and Debug configurations. It also references the file in obj\Release|Debug\Orleans.xml. Again, not sure it that matters, I'm merely looking for consistency.